membership-attribute-service/app/prodtest/FeatureToggleDataUpdatedOnSchedule.scala

```diff
@@ -13,11 +13,11 @@ import scalaz.{-\/, \/-}
 class FeatureToggleDataUpdatedOnSchedule(featureToggleService: ScanamoFeatureToggleService, stage: String)(implicit ec: ExecutionContext, system: ActorSystem) extends LazyLogging {
 
   private val updateZuoraLookupFeatureDataTask: ScheduledTask[ZuoraLookupFeatureData] = {
-    val featureName = "UpdateAttributesFromZuoraLookup"
```

Comment:
I didn't mean for this featureName to diverge from what is is the FeatureToggles dynamo table in the first place. This affects the logging.